src/DDTrace/Processing/TraceAnalyticsProcessor.php

```diff
@@ -13,33 +13,48 @@
  */
 final class TraceAnalyticsProcessor
 {
+    /**
+    * @param bool|float $value
+    */
+    public static function normalizeAnalyticsValue(&$array, $value)
+    {
+        if (true === $value || null === $value) {
```

Comment:
Why should we convert `null` to `100%`? If the reason is, as how I understand it, to [remove this](https://github.com/DataDog/dd-trace-php/pull/398/files#diff-7952f110894c1eceb5170b6255a3550dL42) then I believe we should instead set the metric over there and remove this risky `null === $value` which looks a little bit counter intuitive to me. WDYT?

Reply:
I don't know ;) I only copy pasted it here :D 

Let me take a look :)

Reply:
I agree the `null === $value` looks weird, let me remvoe it and see what breaks.